github-blob.js

```diff
@@ -0,0 +1,75 @@
+class BlobLine {
+  constructor(idx, text) {
+    this.lineNumber = idx;
+    this.text = text.replace(/ /g, "&nbsp;");
+  }
+
+  renderHtml() {
+    return `
+      <tr>
+        <td id="L${this.lineNumber}" class="blob-num js-line-number" data-line-number="${this.lineNumber}"></td>
+        <td id="LC${this.lineNumber}" class="blob-code blob-code-inner js-file-line">${this.text}</td>
+      </tr>
+    `;
+  }
+}
+
+class Blob {
+  constructor(blobLines) {
+    this.blobLines = blobLines;
+  }
+
+  static compute(lines) {
+    const blobLines = lines.split('\n').map((line, idx) => { return new BlobLine(idx, line) });
+    return new Blob(blobLines);
+  }
+
+  static detect() {
+    const header = $('.blob-code:first').text();
+
+    if (header.match(/^\$ANSIBLE_VAULT;.+AES256$/) && !$('#decryptbtn').length) {
+      const container = $('.js-file-line-container');
+      const original = container.html();
+      const textToDecrypt = $('.blob-code').not(':first').text();
```

Comment:
Can you please stick to the existing convention of `underscored_names` for variables so that things stay consistent